src/data/nat/choose.lean

```diff
@@ -1,8 +1,7 @@
 /-
 Copyright (c) 2018 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Chris Hughes
-Mostly based on Jeremy Avigad's choose file in lean 2
```

Comment:
Perhaps this could be added to a module docstring instead of getting deleted entirely?

Reply:
Also, the `Authors` line certainly shouldn't be deleted!

Reply:
Oh, it wasn't deleted – for some reason my comment doesn't show the edit.

Reply:
I moved this to theorem docstrings!

Reply:
Oops, sorry for jumping to conclusions without inspecting the diff properly.

Reply:
I think this comment is out of date anyway, I don't think any of the theorems that are tagged with this are actually from Lean 2, and the ones that are have been moved to `data.nat.basic`.

Reply:
I got rid of my new tags